### PR TITLE
resources: Add Loot Land

### PR DESCRIPTION
**Pull request by VieYang**
## Loot Land

Etherscan link: https://etherscan.io/address/0x8575B7dFE18F4c9A872253bB4f557AD02e94C3A5

Website: https://opensea.io/collection/landproject

Loot Land are parts of the world in Loot metaverse. LAND is divided into parcels that are referenced using unique x,y cartesian coordinates. Feel free to use Loot Land in any way you want.

### CODE REVIEW

**Comment by zhew2013** [comment]
Could you please resolve the conflict?

**Comment by VieYang** [comment]
@zhew2013  ok, all is ok an all checks have passed.

File: utils/lists.ts [hunk 1]
```diff
@@ -462,4 +462,14 @@ export const derivativesList: Record<string, string>[] = [
     description: "Unique 8bit songs for every Loot holder",
     url: "https://www.loottunes.fun/",
   },
+  {
```
Comment (zhew2013) [inline utils/lists.ts:465]: this is not the intended change right? MUDVERSE is already on line 381 above.

**Comment by VieYang** [comment]
@zhew2013  updated

File: utils/lists.ts [hunk 2]
```diff
@@ -462,4 +462,9 @@ export const derivativesList: Record<string, string>[] = [
...
+    name: "Loot Land",
+    description: "Loot Land are parts of the world in Loot metaverse",
+    url: "https://opensea.io/collection/landproject"
```
Comment (zhew2013) [inline utils/lists.ts:468]: Nit: Can you end a comma in the end to match the styling. Otherwise LGTM!

**Review by zhew2013 — CHANGES_REQUESTED** [review]

File: utils/lists.ts (hunk 2 above)
Comment (VieYang) [reply utils/lists.ts]: done.

**Comment by zhew2013** [comment]
All LGTM after you fix the conflict.

**Comment by VieYang** [comment]
sorry, I will close and reopen.

**Pull request closed without merging**